src/System.Management.Automation/engine/SessionStateDriveAPIs.cs

```diff
@@ -963,25 +963,9 @@ private bool IsAStaleVhdMountedDrive(PSDriveInfo drive)
 
                         if (systemDriveInfo.DriveType == DriveType.NoRootDirectory)
                         {
-                            try
-                            {
-                                // Checking for the presence of mounted drive locally using Utils.DirectoryExists API as
-                                // the calls to this API is faster than normal Directory.Exist API.
-                                bool validDrive = Utils.DirectoryExists(drive.Root);
-                                if (!validDrive)
-                                {
-                                    result = true;
-                                }
-                            }
-                            // We don't want to have automounting cause an exception. We
-                            // rather it just fail silently as it wasn't a result of an
-                            // explicit request by the user anyway.
-                            // Following the same pattern as the Calling API.
-                            catch (IOException)
-                            {
-                            }
-                            catch (UnauthorizedAccessException)
+                            if (!Directory.Exists(drive.Root))
```

Comment:
Should this be
```CSharp
return !DirectoryExists(drive.Root)
```

Reply:
Seems original code is easier to understand.